mongo_connector/doc_managers/elastic_doc_manager.py

```diff
@@ -55,8 +57,16 @@ def __init__(self, url, auto_commit_interval=DEFAULT_COMMIT_INTERVAL,
                  unique_key='_id', chunk_size=DEFAULT_MAX_BULK,
                  meta_index_name="mongodb_meta", meta_type="mongodb_meta",
                  attachment_field="content", **kwargs):
+        aws = kwargs.get('aws', {'access_id': '', 'secret_key': '', 'region': 'us-east-1'})
```

Comment:
Let's move this under the `if` below since it doesn't get used unless the user actually specified `aws` in the kwargs.